docs/_articles/en/troubleshooting.md

```diff
@@ -46,7 +46,7 @@ If the Apex Language Server didn’t activate, ensure that you've:
 
 1. Opened a Salesforce DX project that has a valid `sfdx-project.json` file.
 1. Opened the Salesforce DX project as a top-level folder.
-1. Installed Java 8 or Java 11; you’ll see a warning if neither version is installed.
+1. Installed Java 8, Java 11, or Java 17; you’ll see a warning if neither version is installed.
```

Comment:
@sbudhirajadoc I searched the codebase for "Java 8" and updated the docs & strings to include Java 17.  I don't know if this is the copy text you'll want to use, but I thought I'd take a first pass at it, and at least this way, we know which files need updates.